apps/src/templates/sectionAssessments/SectionAssessments.jsx

```diff
@@ -101,6 +107,7 @@ export default connect(state => ({
   assessmentList: getCurrentScriptAssessmentList(state),
   scriptId: state.scriptSelection.scriptId,
   assessmentId: state.sectionAssessments.assessmentId,
+  totalStudentCount: getTotalStudentCount(state),
```

Comment:
It looks like this component only cares about `totalStudentCount` so it can pass it through to child components, but the child components are themselves redux-connected and could retrieve this information themselves.  Thoughts on just letting `MCAssessmentsOverviewContainer` and `MCSurveyOverviewContainer` ask for the total student count directly?